app/controllers/api/v2/smart_proxy_hosts_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Api
+  module V2
+    class SmartProxyHostsController < V2::BaseController
+      before_action :find_proxy
+
+      def resource_class
+        @resource_class ||= Host::Managed
+      end
+
+      def resource_scope(*args)
+        resource_class.authorized(:view_hosts).joins(:infrastructure_facet).merge(::HostFacets::InfrastructureFacet.where(smart_proxy_id: @proxy.id))
+      end
+
+      api :PUT, '/smart_proxies/:smart_proxy_id/hosts/:host_id', N_("Assign a host to the Foreman instance")
+      param :smart_proxy_id, :identifier
+      param :host_id, :identifier
+      def update
+        # TODO?: output
```

Comment:
What is this TODO?

Reply:
Currently when a request to this endpoint succeeds, the body of the response is empty. Left an optional todo there to think if we should try returning something meaningful there

Reply:
I'll interpret the silence here as empty output is fine